lib/trailblazer/test/deprecation/operation/assertions.rb

```diff
@@ -0,0 +1,25 @@
+module Trailblazer::Test
+  module Deprecation
+    module Operation
+      module Assertions
+        include Trailblazer::Test::Operation::Assertions
+
+        # compatibility call for TRB 2.0
+        def ctx(new_params, current_user: nil, default_options: options_pass, **options)
+          final_params = params.merge(new_params)
+          ctx = options_pass.merge(**options)
+          ctx = ctx.merge(current_user: current_user) unless current_user.nil? # otherwise current_user will be always nil
+          [final_params, ctx]
+        end
+
+        def _call_operation(operation_class, *args)
+          operation_class.(args[0][0], args[0][1])
```

Comment:
This could be way better I think, but how?

Reply:
var1, var2 = args[0]
operation_class.(var1, var2)

something like this you wanna?